plugins/content/menu/menu.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Content.menu
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\Registry\Registry;
+
+/**
+ * Menu Plugin
+ *
+ * @since  3.6
+ */
+class PlgUserProfile extends JPlugin
+{
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 * @since  3.6
+	 */
+	protected $autoloadLanguage = true;
+
+	/**
+	 * Adds additional fields to the editing form
+	 *
+	 * @param   JForm  $form  The form to be altered.
+	 * @param   mixed  $data  The associated data for the form.
+	 *
+	 * @return  boolean
+	 *
+	 * @since   3.6
+	 */
+	public function onContentPrepareForm($form, $data)
+	{
+		if (!($form instanceof JForm))
+		{
+			$this->_subject->setError('JERROR_NOT_A_FORM');
+
+			return false;
+		}
+
+		// Check we are manipulating a valid form.
+		$name = $form->getName();
+
+		if (!in_array($name, array('com_content.article')))
+		{
+			return true;
+		}
+
+		// Add the registration fields to the form.
+		JForm::addFormPath(__DIR__ . '/menus');
```

Comment:
Name form forder to `forms` instead of `menus`. Because ultimately they are forms, not menu.